ui/apps/platform/src/services/IntegrationHealthService.js

```diff
@@ -21,5 +21,10 @@ export const fetchPluginIntegrationsHealth = () =>
         .get('/v1/integrationhealth/notifiers')
         .then((response) => response?.data?.integrationHealth ?? []);
 
+export const fetchDeclarativeConfigurationsHealth = () =>
+    axios
+        .get('/v1/integrationhealth/declarativeconfigs')
```

Comment:
Not a FE fix, but I think we should advocate for more human-readable endpoints, like `integration-health/declarative-configs`

Reply:
I understand why we can't change existing, but there is discussion about starting a `v2` of the API. Maybe we can't fix them all in the new iteration.